src/Compilers/Core/CodeAnalysisTest/Analyzers/AnalyzerConfigTests.cs

```diff
@@ -177,15 +177,18 @@ public void SpacesInProperties()
                 properties);
         }
 
-        [Fact]
-        public void EndOfLineComments()
+        [Theory]
```

Comment:
Please document the breaking change in https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20post%20DotNet%205.md

Reply:
➡️ Documented for .NET 9